### PR TITLE
Technical Components and Items Category addition.

### PR DESCRIPTION
**Pull request by BurningBrimstone**
This pull request adds a page for the list of Technical Components aswell as adds each individual technical component.

The Items Category was also added.

Minor changes were made to the syntax of other pages to ease reading.

### CODE REVIEW

File: pages/Backpacks.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The page is work in progress.
```
Comment (TheBusyBiscuit) [inline pages/Backpacks.md]: Just no.

File: pages/Cooler.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The Cooler is an [Item](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Items) that has a unique use.
+
+## Obtaining
+The Cooler can be crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+The Portable Crafter can be used while holding the item by right clicking. Upon right clicking, the item will open up a chest GUI in which the player can place any [Juice](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Juices). If the cooler is in the player's inventory, and the player is hungry, the cooler will automatically feed the player the juices inside.
```
Comment (TheBusyBiscuit) [inline pages/Cooler.md]: Portable Crafter?

File: pages/Hardened-Glass.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Hardened Glass is a [Technical Component](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Technical-Components) that can be used to craft a variety of other Technical Components.
+
+## Obtaining
+Hardened Glass can be crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table)
+
+## Special Properties
+Hardened Glass is resistant to most explosions.
```
Comment (TheBusyBiscuit) [inline pages/Hardened-Glass.md]: Maybe mention that it is not resistant to Withers, but the wither-proof glass is , in a [see also].

File: pages/Items.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+The Items category contains various items that all have their own unique use.
+
+## Table of Contents
+* [Portable Crafter](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Portable-Crafter)
+* [Portable Dustbin](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Portable-Dustbin)
+* [Rag](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Bandage](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Splint](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Vitamins](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Medicine](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
```
Comment (TheBusyBiscuit) [inline pages/Items.md]: Since this page will host all medical supplies, it should rather be called "Medical-Supplies", not "Medical-Supply"

File: pages/Medical-Supply.md [hunk 5]
```diff
@@ -0,0 +1 @@
+## Page is work in progress.
```
Comment (TheBusyBiscuit) [inline pages/Medical-Supply.md]: Don't do that.

File: pages/Portable-Dustbin.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+The Portable Dustbin is an [Item](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Items) that has a unique use.
+
+## Obtaining
+The Portable Dustbin can be crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+The Portable Dustbin can be used while holding the item by right clicking. Upon right clicking, the item will open up a chest GUI in which any item can be placed, upon the player closing the GUI by pressing ESC, any items placed in the GUI will be deleted and unobtainable.
```
Comment (TheBusyBiscuit) [inline pages/Portable-Dustbin.md]: Maybe not mention the key ESC.
Keys can be re-bound and there are many more ways to close an inventory.
Some smartass may think "Oh but the wiki said it'd only do it if I press this specific key.

Comment (TheBusyBiscuit) [inline pages/Portable-Dustbin.md]: Unobtainable does not sound right to me.
Maybe reword it to say "they will be lost forever, so be careful".

File: pages/Steel-Thruster.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+The Magnet is a [Technical Component](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Technical-Components) that can be used to craft [Jetpacks](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Jetpacks) and [Jet Boots](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Jet-Boots).
```
Comment (TheBusyBiscuit) [inline pages/Steel-Thruster.md]: Magnet?

File: pages/Backpacks.md (hunk 1 above)
Comment (BurningBrimstone) [reply pages/Backpacks.md]: Resolved.

File: pages/Cooler.md (hunk 2 above)
Comment (BurningBrimstone) [reply pages/Cooler.md]: Resolved.

File: pages/Items.md (hunk 4 above)
Comment (BurningBrimstone) [reply pages/Items.md]: Resolved.

File: pages/Medical-Supply.md (hunk 5 above)
Comment (BurningBrimstone) [reply pages/Medical-Supply.md]: Resolved.

File: pages/Portable-Dustbin.md (hunk 6 above)
Comment (BurningBrimstone) [reply pages/Portable-Dustbin.md]: Changed to "lost forever. For this reason, players should be cautious when using this item."
Resolved.

File: pages/Steel-Thruster.md (hunk 7 above)
Comment (BurningBrimstone) [reply pages/Steel-Thruster.md]: Resolved.

File: pages/Portable-Dustbin.md (hunk 6 above)
Comment (BurningBrimstone) [reply pages/Portable-Dustbin.md]: Simply removed "by pressing ESC"
Resolved.

File: pages/Hardened-Glass.md (hunk 3 above)
Comment (BurningBrimstone) [reply pages/Hardened-Glass.md]: Added that it is not immune to wither explosions, also added a See Also to Wither-Proof Blocks.
